region/brinstar/kraid/Kraid Eye Door Room.json

```diff
@@ -290,6 +309,109 @@
         "A beam shot, Missile, or Super can be used."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Leave Spinning (Space Jump, Door Open)",
+      "requires": [
+        {"doorUnlockedAtNode": 1},
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 5,
+            "openEnd": 1
+          }
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 3],
+      "name": "Leave With Mockball (Space Jump, Door Open)",
+      "requires": [
+        {"doorUnlockedAtNode": 1},
+        "SpaceJump"
```

Comment:
These are similar to `canPreciseSpaceJump`, but you are going slow with a health refill right here for when you hit the spikes.
So canTrickyJump?